pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controls.go

```diff
@@ -34,6 +34,42 @@ func (s ResourceStatus) String() string {
 	return names[s]
 }
 
+func Namespace(n NFD) (ResourceStatus, error) {
+
+	state := n.idx
```

Comment:
Your variable names are not ideal, especially the NFD struct: rec, ins, idx. I don't know what any of these are at a glance, see: [variable names](https://github.com/golang/go/wiki/CodeReviewComments#variable-names).

Reply:
Lets keep that in mind for the next release.

Reply:
Lets keep this in mind for the next release.